lib/oli_web/live/community_live/overview_row_component.ex

```diff
@@ -0,0 +1,22 @@
+defmodule OliWeb.CommunityLive.OverviewRowComponent do
```

Comment:
https://github.com/wyeworks/oli-torus/blob/feature/MER-408-disable-community/lib/oli_web/live/community_live/show_section_component.ex

![image](https://user-images.githubusercontent.com/26532202/141514521-256f65cf-6e51-460e-8e9c-753348df731c.png)

Reply:
😂  

I'll rebase my branch with the disable community and reuse that component.